### ISSUE TITLE
I have noticed that if jquery ui is loaded the jquery resizable does not work.

### ISSUE THREAD

**Issue by handysolver** [RickStrahl/jquery-resizable#25]

**Comment by Sulstice** [comment]
Yeah I ran into this problem as well and kind of blocking the use of this library

**Comment by morissetcl** [comment]
Same here.

**Comment by pmorch** [comment]
I'm guessing it is because jQueryUI also has a [$().`resizable()`](http://jqueryui.com/resizable/) method.

**Comment by RickStrahl** [comment]
Definitely true...  but if you're using jquery-ui you certainly don't need this plugin. It was created to avoid for the use case where a larger framework isn't available.

**Comment by thiscafer** [comment]
How do you use with  jquery ui?

**Comment by dominicgan** [comment]
@thiscafer you can rename the function to make it work alongside jquery ui.

**Comment by taleteller** [comment]
Actually there is use of this method alongside `jquery-ui`, it allows me to specify and style an custom divider handle. I renamed it for myself as `resizer` but I would prefer to use the original npm package.

**Comment by RickStrahl** [comment]
Revisiting this after some thought I think it would be pretty easy to add a `.resizableSafe()` alias that would allow any components that use `jQuery.resizable()` to get a reliable reference to the plugin.

If made some changes for 0.35+ that do:


```js
if ($.fn.resizableSafe)
        return;

$.fn.resizableSafe = function fnResizable(options) {
     ... main implementation
}

if (!$.fn.resizable)
   $.fn.resizable = $.fn.resizableSafe;
```

You should then be able to replace any reference to `.resizable()` with `.resizableSafe()` to co-exist with jquery.ui or any other component that also use `.resizable()`.
